website/routes.py

```diff
@@ -427,13 +427,6 @@ def make_url_map(app):
         ),
 
         Rule(
-            '/resend/',
```

Comment:
@chennan47 Just a heads up--turns out we actually do need this route. This is used for resending confirmation emails after signing up. The "Resend" link on the CAS login page links to it.

I'll work on readding this.